Dockerfile

```diff
@@ -1,3 +1,26 @@
-FROM klakegg/hugo:ext-alpine
+FROM registry.fedoraproject.org/fedora-minimal
 
-RUN apk add git
+ARG HUGO_VERSION
+
+VOLUME /site
+WORKDIR /site
+
+# Avoid startup errors about missing AsciiDoctor
+ARG ASCIIDOCTOR_VERSION=2.0.10
+RUN microdnf -y install ruby
+RUN gem install --no-document "asciidoctor:${ASCIIDOCTOR_VERSION}"
```

Comment:
Are you sure we need this?

Reply:
I get startup warnings otherwise

Reply:
Personally, I prefer removing asciidoctor to effectively ban usage of adoc files.

I ran `find -iname '*.adoc'` on the kiali.io code. It returned this file list:

```
./README.adoc
./content/en/docs/Architecture/terminology/_index.adoc
./content/en/docs/Tutorials/travels/99-uninstall/01-uninstall-travel-demo.adoc
```

I think converting those two adoc files to markdown should remove the need of asciidoctor.

Reply:
That's OK by me. That should not block this PR though. When we submit a PR to remove those adoc files, we can then attempt to rip it out of the container.